pyproject.toml

```diff
@@ -185,7 +175,7 @@ markers = [
 ]
 
 [tool.uv]
-default-groups = ["dev"]
+default-groups = ['dev']
```

Comment:
Did you want to add a lint step to precommit that helps enforce standardization here?

Reply:
Yes tried to find something without success, I'll look a bit more

Reply:
There's https://github.com/tox-dev/toml-fmt but way too much diff generated with our current file. For now let's just be careful when making edits to the pyproject.toml